activerecord/lib/active_record/relation/query_methods.rb

```diff
@@ -852,7 +852,7 @@ def none! # :nodoc:
       where!("1=0").extending!(NullRelation)
     end
 
-    # Sets readonly attributes for the returned relation. If value is
+    # Sets the returned relation to readonly mode. If value is
```

Comment:
I kept this docs change because the current version is misleading; "readonly attributes" are a different feature:

https://api.rubyonrails.org/classes/ActiveRecord/ReadonlyAttributes/ClassMethods.html